cmake/CodeCoverage.cmake

```diff
@@ -1,3 +1,4 @@
+# cmake-format: off
```

Comment:
Good call.  I believe we're in the right to do it this way and I can't find a convenient way to ignore certain files from a cmake-format configuration file.